extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
@choumx can correct me if I'm wrong but I don't think we're going to support all of those attributes. For instance the `on` attribute is used to communicate state changes to bind; allowing that itself to be a bind attribute could get pretty hairy.